transport/websocket.go

```diff
@@ -87,11 +88,13 @@ type WebsocketTransport struct {
 	ReceiveTimeout time.Duration
 	SendTimeout    time.Duration
 
+	InsecureTls bool
```

Comment:
[Initialisms should be capitalized.](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)